test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -271,6 +271,89 @@ function ExecuteRequestWithCustomUserAgent {
     return $result
 }
 
+function GetSelfSignedCert {
```

Comment:
@TravisEz13 I tested with that. But the cert in `New-GoodCertificate` does not contain the proper key usage to be used for SSL/TLS Auth. It is issued for only Document Encryption. I was afraid if I change the cert in `certificateCommon.psm1` I might be break tests that depend on that restriction. Thus, I added a new function in  `WebCmdlets.Tests.ps1` instead. What do you suggest?